src/mutations/user/walletLogin.ts

```diff
@@ -169,6 +169,7 @@ const resolver: MutationToWalletLoginResolver = async (
   // Signup otherwise
   const newUser = await userService.create({
     userName: ethAddress,
+    displayName: ethAddress,
```

Comment:
how about `0x123...abc` as default userName and displayName?

 cc @devformatters